admin_manual/configuration_server/caching_configuration.rst

```diff
@@ -30,19 +29,79 @@ of memcache that best fits your needs. The supported caching backends are:
 * `Memcached <https://www.memcached.org/>`_
    For distributed caching.
    
-Memcaches must be explicitly configured in Nextcloud by installing
+Data caches, or memcaches, must be explicitly configured in Nextcloud by installing
 and enabling your desired cache, and then adding the appropriate entry to 
 ``config.php`` (See :doc:`config_sample_php_parameters` for an overview of
 all possible config parameters).
 
+
+Recommendations based on type of deployment
+-------------------------------------------
+
 You may use both a local and a distributed cache. Recommended caches are APCu 
-and Redis. After installing and enabling your chosen memcache, verify that it is 
-active by running :ref:`label-phpinfo`.
+and Redis. After installing and enabling your chosen memcache (data cache),
+verify that it is active by running :ref:`label-phpinfo`.
+
+.. note:: See specific cache configuration options under the appropriate section further down.
+
+Small/Private home server
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Only use APCu::
+
+    'memcache.local' => '\OC\Memcache\APCu',
```

Comment:
also set distributed, because both may be used, and the use case is a different one.

Reply:
@blizzz also this one has been there [before](https://github.com/nextcloud/documentation/pull/9093/files#diff-48dc187eeb03371b5ad6886b33043bf060884809c1d51a3568c5d29e40e3bf13L221-L227).